### PR TITLE
deploy dashboard

### PR DESCRIPTION
**Pull request by izhuxiaoqing**

### CODE REVIEW

File: docs-2.0/nebula-dashboard/deploy-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+```
+
+## Directory
+
+Run `tar -xvf {{dashboard.release}}.tar.gz` to decompress the installation package. There are 5 subdirectories in the `nebula-graph-dashboard`. The descriptions are as follows.
+
+|Name|Description|
+|:---|:---|
+|node-exporter | Collects the source information of machines in the cluster, including the CPU, memory, load, disk, and network.|
+|nebula-stats-exporter | Collects the performance metrics in the cluster, including the IP addresses, versions, and monitoring metrics (such as the number of queries, the latency of queries, the latency of heartbeats, and so on.|
```
Comment (whitewum) [inline docs-2.0/nebula-dashboard/deploy-dashboard.md:34]: `)` missing

File: docs-2.0/nebula-dashboard/deploy-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        endpointPort: 19779
+  ```
+
+2. Return the following statement to start the service:
+
+  ```bash
+  $ nohup ./nebula-stats-exporter --bare-metal --bare-metal-config=./config.yaml &
+  ```
+
+After the service is started, you can return `<IP>:9100` in the browser to check whether the service is started normally.
```
Comment (whitewum) [inline docs-2.0/nebula-dashboard/deploy-dashboard.md:103]: return ?  enter

File: docs-2.0/nebula-dashboard/deploy-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
+    You only need to deploy the `nebula-http-gateway` service on the machine where the `nebula-graph-dashboard` service is installed.
+
+To start the service, return the following statement in `nebula-http-gateway`:
+
+```bash
+$ nohup ./nebula-httpd &
+```
+
+After the service is started, you can return `<IP>:8090` in the browser to check whether the service is started normally.
```
Comment (whitewum) [inline docs-2.0/nebula-dashboard/deploy-dashboard.md:154]: ditto

**Review by whitewum — APPROVED** [review]

File: docs-2.0/nebula-dashboard/deploy-dashboard.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
+# Deploy Dashboard
+
+The installation and deployment of Dashboard involve five kinds of services. This topic will describe how to install and deploy Dashboard in detail.
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard/deploy-dashboard.md:3]: Deployment usually includes installation. So no need to use both.

File: docs-2.0/nebula-dashboard/deploy-dashboard.md [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+
+|Name|Description|
+|:---|:---|
+|node-exporter | Collects the source information of machines in the cluster, including the CPU, memory, load, disk, and network.|
+|nebula-stats-exporter | Collects the performance metrics in the cluster, including the IP addresses, versions, and monitoring metrics (such as the number of queries, the latency of queries, the latency of heartbeats, and so on).|
+|prometheus | The time series database that stores monitoring data. |
+|nebula-http-gateway | Provides HTTP ports for cluster services to access the prometheus service or execute nGQL statements to interact with the Nebula Graph database.|
+|nebula-graph-dashboard| Provides the Dashboard service. Note that the name of this directory is the same as the root directory. The following `nebula-graph-dashboard` refers to the subdirectory.|
+
+The above five directories need to be duplicated to each machine according to deployment needs. The details are as follows.
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard/deploy-dashboard.md:39]: Ambiguous. Please clarify on which machines must the directories be placed.

File: docs-2.0/nebula-dashboard/deploy-dashboard.md [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+
+## Procedure
+
+### How to deploy the `node-exporter` service
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard/deploy-dashboard.md:43]: Why not use the verb + object form like `## Download Dashboard`? Consistency is important in technical writing.

File: docs-2.0/nebula-dashboard/deploy-dashboard.md [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+      - instanceName: storaged1
+        endpointIP: 192.168.xx.6
+        endpointPort: 19779
+        componentType: storaged
+      - instanceName: storaged2
+        endpointIP: 192.168.xx.7
+        endpointPort: 19779
+  ```
+
+2. Return the following statement to start the service:
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard/deploy-dashboard.md:97]: Run?

File: docs-2.0/nebula-dashboard/deploy-dashboard.md [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+        - targets: [
+            '192.168.xx.101:9200'  # The IP address and port of the node-exporter service.
+          ]
+  ```
+
+  - scrape_interval: The interval for collecting the monitoring data, which is 1 minute by default.
+
+  - evaluation_interval: The interval for running alarm rules, which is 1 minute by default.
+
+2. Return the following statement to start the service.
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard/deploy-dashboard.md:134]: Same as above.

File: docs-2.0/nebula-dashboard/deploy-dashboard.md [hunk 9]
```diff
@@ -0,0 +1,195 @@
...
+          "ip:port": "instance1"
+      },
+      "chartBaseLine": {
+
+      }
+  }
+  ...
+  ```
+
+2. To start the service, return the following statement in `nebula-graph-dashboard`:
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard/deploy-dashboard.md:176]: Same as above. Check through the topic.

**Review by randomJoe211 — DISMISSED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Review by foesa-yang — APPROVED** [review]

**Pull request merged**